src/snowflake/cli/api/constants.py

```diff
@@ -38,6 +38,9 @@ class ObjectType(Enum):
     USER = ObjectNames("user", "user", "users")
     WAREHOUSE = ObjectNames("warehouse", "warehouse", "warehouses")
     VIEW = ObjectNames("view", "view", "views")
+    IMAGE_REPOSITORY = ObjectNames(
+        "image-repository", "image repository", "image repositories"
+    )
```

Comment:
```
@app.command(
    help=f"Provides description of an object of given type. {SUPPORTED_TYPES_MSG}"
)
@with_output
@global_options_with_connection
def describe(
```

Help still will be printing image repository as supported object for describe. You should change list of objects in help.